tests/integration/test_requirements_dev.py

```
@@ -257,14 +273,209 @@
+    
+    def test_all_packages_use_consistent_operators(self):
+        """Test that version constraints use consistent comparison operators (prefer >=)."""
+        assert REQUIREMENTS_FILE.exists(), "requirements-dev.txt not found"
+        
+        requirements = parse_requirements(REQUIREMENTS_FILE)
+        
+        # Count operator usage using regex pattern that matches operators followed by version
+        # This avoids substring matching issues (e.g., >= being counted as both >= and >)
+        operator_counts = {'>=': 0, '==': 0, '<=': 0, '>': 0, '<': 0, '~=': 0}
+        
+        for pkg, version_spec in requirements:
```

Ruff (0.14.6)

Loop control variable `pkg` not used within loop body

Rename unused `pkg` to `_pkg`

(B007)